scripts/bundle_creation.sh

```diff
@@ -85,27 +88,32 @@ airgap_bundle_create() {
   args+=("${original_aib_path}")
   # printf '%s\n' "Running: ${CHEF_AUTOMATE_BIN_PATH} ${args[*]}"
   if "${CHEF_AUTOMATE_BIN_PATH}" "${args[@]}" > /tmp/thelog.log; then
+    BUILD_VERSION=`"${CHEF_AUTOMATE_BIN_PATH}" airgap bundle info "${original_aib_path}" | grep -i version | awk '{print $2}'`
     if [ "$BUNDLE_TYPE" == "upgradefrontends" ] || [ "$BUNDLE_TYPE" == "all" ]
     then
-          cat "${original_aib_path}" > "${TARBALL_PATH}"
-          outfile=${ORIGINAL_TARBALL:-${TARBALL_PATH}}
+          frontend_aib="/hab/a2_deploy_workspace/terraform/transfer_files/frontend-${BUILD_VERSION}.aib"
```

Comment:
frontend_aib="${terraform path what we will get from makefile} transfer_files/frontend-${BUILD_VERSION}.aib"